specs/ukhpi-data-query.spec

```diff
@@ -0,0 +1,125 @@
+# UKHPI application
+
+Tags: UKHPI
+
+The UK house price index displaying value of house sales
+
+## Landing page
+
+Tags: Landing, root
+
+* The page loads with appropriate body content
+* The page contains multiple links to the applications
+* The page contains multiple links to external websites
+
+## Change language
+
+Tags: locale
+
+Users can toggle between English and Welsh language
+
+* Click Cymraeg link in top right
+* Displayed body copy will change to Welsh language
+* Navigation items will be displayed in Welsh language
+* Clicking main navigation title will route user back to home page and language will default to English
+* Click English link in top right
+* All Displayed text will change to English language
```

Comment:
With the possible exception of some place names, which don't have Welsh-language equivalents